plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java

```diff
@@ -139,7 +139,8 @@ private String extractArchive(String fileName, File outputDirectory) {
 		String extractedDirectory = null;
 		if (fileName.endsWith(SUFFIX_ZIP)) {
 			extractedDirectory = StringUtils.chomp(fileName, SUFFIX_ZIP);
-			FileHelper.unzipFile(new File(fileName), outputDirectory);
+			FileHelper.createDirectory(new File(extractedDirectory));
```

Comment:
looking at the source of FileHelper, your change seems required. I believe that this should be fixed in FileHelper though, making sure unzip and un-targz behave the same - creating the destination directory if it doesnt.